README.md

```diff
@@ -143,8 +153,11 @@ recognition.finished?
 
 Request fashion coordinates generation.
 
+Refer: [`POST /v1/coordinates`](https://api.scnnr.cubki.jp/v1/docs#tag/coordinates%2Fpaths%2F~1coordinates%2Fpost)
+
 ```
-coordinate = client.coordinate('tops', ['グレー', 'パーカー'], casual: 0.7, girly: 0.3)
+tastes = { casual: 0.7, girly: 0.3 }
+coordinate = client.coordinate('tops', ['グレー', 'パーカー'], tastes, target: 2)
```

Comment:
I feel this example is inconsistent a little bit because `target: 2` is specified but the response has 3 items. 
Should we update the example?

Reply:
Ah, I overlooked it... Thank you for pointing out!